tests/system-tests/internal/apiobjectshelper/apiobjectshelper.go

```diff
@@ -81,3 +89,251 @@ func VerifyOperatorDeployment(apiClient *clients.Settings,
 
 	return nil
 }
+
+// CreateServiceAccount creates the service account and verifies it was created.
+func CreateServiceAccount(apiClient *clients.Settings, saName, nsName string) error {
+	glog.V(100).Infof(fmt.Sprintf("Creating ServiceAccount %q in %q namespace",
+		saName, nsName))
+	glog.V(100).Infof("Creating SA %q in %q namespace", saName, nsName)
+
+	deploySa := serviceaccount.NewBuilder(apiClient, saName, nsName)
+
+	err := wait.PollUntilContextTimeout(
+		context.TODO(),
+		time.Second*15,
+		time.Minute,
+		true,
+		func(ctx context.Context) (bool, error) {
+			deploySa, err := deploySa.Create()
+
+			if err != nil {
+				glog.V(100).Infof("Error creating SA %q in %q namespace: %v", saName, nsName, err)
+
+				return false, nil
+			}
+
+			glog.V(100).Infof("Created SA %q in %q namespace",
+				deploySa.Definition.Name, deploySa.Definition.Namespace)
+
+			return true, nil
+		})
+
+	if err != nil {
+		return fmt.Errorf("failed to create ServiceAccount %q in %q namespace", saName, nsName)
+	}
+
+	return nil
+}
+
+// CreateClusterRBAC creates the RBAC and verifies it was created.
+func CreateClusterRBAC(
+	apiClient *clients.Settings,
+	rbacName, clusterRole, saName, nsName string) error {
+	glog.V(100).Infof("Creating RBAC for SA %s", saName)
+
+	glog.V(100).Infof("Creating ClusterRoleBinding %q", rbacName)
+	crbSa := rbac.NewClusterRoleBindingBuilder(
+		apiClient,
+		rbacName,
+		clusterRole,
+		rbacv1.Subject{
+			Name:      saName,
+			Kind:      "ServiceAccount",
+			Namespace: nsName,
+		})
+
+	err := wait.PollUntilContextTimeout(
+		context.TODO(),
+		time.Second*15,
+		time.Minute,
+		true,
+		func(ctx context.Context) (bool, error) {
+			crbSa, err := crbSa.Create()
+			if err != nil {
+				glog.V(100).Infof(
+					"Error Creating ClusterRoleBinding %q : %v", crbSa.Definition.Name, err)
+
+				return false, nil
+			}
+
+			glog.V(100).Infof("ClusterRoleBinding %q created:\n\t%v",
+				crbSa.Definition.Name, crbSa)
+
+			return true, nil
+		})
+
+	if err != nil {
+		return fmt.Errorf("failed to create ClusterRoleBinding '%s' during timeout %v; %w",
+			rbacName, time.Minute, err)
+	}
+
+	return nil
+}
+
+// DeleteService deletes the service and verifies it was removed.
+func DeleteService(apiClient *clients.Settings, svcName, nsName string) error {
+	glog.V(100).Infof("Delete service %q from nsname %s", svcName, nsName)
```

Comment:
nit: when logging better to use `namespace` and not _nsname_
```suggestion
	glog.V(100).Infof("Delete service %q from namespace %s", svcName, nsName)
```